### PR TITLE
[TASK] avoid duplicate content on extensions.typo3.org and docs.typo3…

### PR DESCRIPTION
**Pull request by sbusemann**
….org

Update page "Publish an Extension"

Add links from extensions.typo3.org to docs.typo3.org and moved content

resolves: https://gitlab.typo3.org/services/t3o-sites/extensions.typo3.org/ter/-/issues/383

### CODE REVIEW

File: Documentation/ExtensionArchitecture/PublishExtension/PublishToTER/Index.rst [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+* Did you read and understand the `TYPO3 Extension Security Policy <https://typo3.org/community/teams/security/extension-security-policy>`__?
+* Does your extension include or need external libraries? Watch for the license! Learn more about the `right licensing <https://typo3.org/project/licenses/>`__.
+* Do you have a public repository on e.g. GitHub, Gitlab or Bitbucket?
+* Do you have the resources to maintain this extension?
+* This means that you should
+
+  * support users and integrators using your extension
+  * review and test contributions
+  * test your extension for new TYPO3 releases
+  * provide and update a documentation for your extension
```
Comment (gilbertsoft) [inline Documentation/ExtensionArchitecture/PublishExtension/PublishToTER/Index.rst:28]:
```suggestion
*  What additional benefit does your extension have for the TYPO3 community?
*  Does your extension key describe the extension? See the extension key requirements.
*  Are there any extensions in the TER yet which have the same functionalities?
*  If yes, why do we need your one? Wouldn't it be an option to contribute to other extensions?
*  Did you read and understand the `TYPO3 Extension Security Policy <https://typo3.org/community/teams/security/extension-security-policy>`__?
*  Does your extension include or need external libraries? Watch for the license! Learn more about the `right licensing <https://typo3.org/project/licenses/>`__.
*  Do you have a public repository on e.g. GitHub, Gitlab or Bitbucket?
*  Do you have the resources to maintain this extension?
*  This means that you should

   *  support users and integrators using your extension
   *  review and test contributions
   *  test your extension for new TYPO3 releases
   *  provide and update a documentation for your extension
```

File: Documentation/ExtensionArchitecture/PublishExtension/PublishToTER/Index.rst [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+Use semantic versions
+=====================
+
+We would like you to stick to semantic versions.
+
+Given a version number **MAJOR.MINOR.PATCH**, increment the:
+
+* **MAJOR** version when you make incompatible API changes (known as "**breaking changes**"), 
+* **MINOR** version when you **add functionality in a backwards-compatible manner**, and
+* **PATCH** version when you **make backwards-compatible bug fixes**.
```
Comment (gilbertsoft) [inline Documentation/ExtensionArchitecture/PublishExtension/PublishToTER/Index.rst:39]:
```suggestion
*  **MAJOR** version when you make incompatible API changes (known as "**breaking changes**"), 
*  **MINOR** version when you **add functionality in a backwards-compatible manner**, and
*  **PATCH** version when you **make backwards-compatible bug fixes**.
```

File: Documentation/ExtensionArchitecture/PublishExtension/PublishToTER/Index.rst [hunk 3]
```diff
@@ -0,0 +1,65 @@
...
+How to publish an extension
+===========================
+
+Now we come to the process of publishing. You have two possibilites to release an extension:
+
+#. Via the web form on extensions.typo3.org (click on "Publish" next to your extension key in the `extension key management <https://extensions.typo3.org/my-extensions>`__).
+#. Via `Tailor <https://github.com/TYPO3/tailor>`__: Tailor is a CLI application to help you maintain your extensions. Tailor talks with the TER REST API and enables you to register new keys, update extension information and publish new versions to the extension repository. (recommended)
+#. Via the SOAP interface using a tool like the `TYPO3 Repository Client <https://github.com/NamelessCoder/typo3-repository-client>`__ (`Documentation <https://github.com/NamelessCoder/typo3-repository-client#typo3-repository-client-apicli>`__) or the `TER client <https://github.com/helhum/ter-client>`__ (`Documentation <https://github.com/helhum/ter-client#ter-client>`__) (deprecated)
+
+
```
Comment (gilbertsoft) [inline Documentation/ExtensionArchitecture/PublishExtension/PublishToTER/Index.rst:65]:
```suggestion
```

**Review by gilbertsoft — CHANGES_REQUESTED** [review]

File: Documentation/ExtensionArchitecture/PublishExtension/PublishToTER/Index.rst [hunk 4]
```diff
@@ -0,0 +1,63 @@
...
+#. A possibility to get in contact with you (link to an issue tracker like forge, GitHub, etc.)
+#. A possibility to look into the code (link to a public repository)
+
+You can edit these options in the `extension key management <https://extensions.typo3.org/my-extensions>`__ (after login)
+
+How to publish an extension
+===========================
+
+Now we come to the process of publishing. You have two possibilites to release an extension:
```
Comment (alexander-nitsche) [inline Documentation/ExtensionArchitecture/PublishExtension/PublishToTER/Index.rst:59]: > two possibilites

three possibilities

File: Documentation/ExtensionArchitecture/PublishExtension/PublishToTER/Index.rst [hunk 5]
```diff
@@ -0,0 +1,63 @@
...
+
+#. Via the web form on extensions.typo3.org (click on "Publish" next to your extension key in the `extension key management <https://extensions.typo3.org/my-extensions>`__).
+#. Via `Tailor <https://github.com/TYPO3/tailor>`__: Tailor is a CLI application to help you maintain your extensions. Tailor talks with the TER REST API and enables you to register new keys, update extension information and publish new versions to the extension repository. (recommended)
```
Comment (alexander-nitsche) [inline Documentation/ExtensionArchitecture/PublishExtension/PublishToTER/Index.rst:62]: would add a minimal CLI example here

Comment (alexander-nitsche) [inline Documentation/ExtensionArchitecture/PublishExtension/PublishToTER/Index.rst:62]: would add a link to REST API, e.g. https://extensions.typo3.org/faq/rest-api/

File: Documentation/ExtensionArchitecture/PublishExtension/PublishToTER/Index.rst [hunk 6]
```diff
@@ -0,0 +1,63 @@
...
+#. Via the SOAP interface using a tool like the `TYPO3 Repository Client <https://github.com/NamelessCoder/typo3-repository-client>`__ (`Documentation <https://github.com/NamelessCoder/typo3-repository-client#typo3-repository-client-apicli>`__) or the `TER client <https://github.com/helhum/ter-client>`__ (`Documentation <https://github.com/helhum/ter-client#ter-client>`__) (deprecated)
```
Comment (alexander-nitsche) [inline Documentation/ExtensionArchitecture/PublishExtension/PublishToTER/Index.rst:63]: would remove linking here, to at most 2, to reduce importance of this list item

Comment (susannemoog) [reply Documentation/ExtensionArchitecture/PublishExtension/PublishToTER/Index.rst:63]: Can you add a PR for your changes and describe what you did for the Tailor part?

Comment (alexander-nitsche) [reply Documentation/ExtensionArchitecture/PublishExtension/PublishToTER/Index.rst:63]: Ok. See https://github.com/TYPO3-Documentation/TYPO3CMS-Reference-CoreApi/issues/1420.

**Pull request merged**